assemblies/assembly_integrating-dev-studio/main.adoc

```diff
@@ -12,6 +12,11 @@ include::_artifacts/author-group.adoc[]
 // Purpose statement for the assembly
 As a developer, you can integrate Red Hat Developer Studio with {PRODUCT} to write complex applications and leverage code auto-completion.
 
+[WARNING]
+====
+Red Hat Developer Studio is in maintenance mode. Therefore, Installing and setting up the development environment with Red Hat Developer Studio for {PRODUCT} {PRODUCT_VERSION} is deprecated.
```

Comment:
@csherrar The "therefore" doesn't fit imo. It's not deprecated because it's in maintenance mode. It's in maintenance mode and is deprecated. And it's not the installing and setting up of it that's deprecated, it's the product itself.

I'd just say this:

"Red Hat Developer Studio is in maintenance mode and is no longer fully supported by Red Hat."

@tomasdavidorg, do you agree?

Reply:
@sterobin I will adopt this phrasing and add anything else @tomasdavidorg would like to add or correct.

Reply:
This one looks better. Thanks for the update.